lxd/containers.go

```diff
@@ -757,27 +757,20 @@ func (d *lxdContainer) applyConfig(config map[string]string) error {
 	}
 
 	if lxcConfig, ok := config["raw.lxc"]; ok {
-		for _, line := range strings.Split(lxcConfig, "\n") {
-			contents := strings.SplitN(line, "=", 2)
-			key := contents[0]
-			value := ""
-			if len(contents) > 1 {
-				value = contents[1]
-			}
-
-			key = strings.Trim(key, " ")
-			value = strings.Trim(value, " ")
-
-			// empty lines
-			if key == "" {
-				continue
-			}
+		f, err := ioutil.TempFile("", "lxd_config")
```

Comment:
indeed... I wish things were a bit more consistent...

Personally I'm in favor of the trailing underscore as a separator and of using the same kind of naming everywhere, so the TempDir test ones should probably become lxd_test_source_ and lxd_test_sink_ or something like that.

Anyway, bikeshedding :)